radio/src/gui/128x64/view_main.cpp

```diff
@@ -47,15 +47,15 @@
 #define TRIM_LH_POS   (TRIM_LH_X-4*FW)
 #define TRIM_RH_NEG   (TRIM_RH_X+1*FW)
 #define TRIM_RH_POS   (TRIM_RH_X-4*FW)
-#if defined(TELEMETRY_FRSKY)
+//#if defined(TELEMETRY_FRSKY)
```

Comment:
Can be removed instead of commenting out because that's how it is in current OpenTX.